packages/core-modules/contracts/storage/ElectionStorage.sol

```diff
@@ -38,6 +38,22 @@ contract ElectionStorage {
          * @dev Position of an address on the nominees Array.
          */
         mapping(address => uint256) nomineeIndexes;
+        /**
+         * @dev Flag to indicate if the election was evaluated (if batched, latest batch was processed).
+         */
+        bool isElectionEvaluated;
+        /**
+         * @dev Used to keep track of the next epoch's winners. Gets erased when an epoch starts and a new council takes effect.
+         */
+        address[] winners;
+        /**
+         * @dev Used to keep track of the next epoch's winners votes. Gets erased when an epoch starts and a new council takes effect.
+         */
+        uint256[] winnersVotes;
```

Comment:
Pls review english grammar in some of the variables in ElectionStorage for this rule: You usually don't use two plurals in the same sentences. I.e. winnersVotes 👉 winnerVotes. Pointed out some of these in other PRs, and I'm not going to point out every case in this one (or check). Please check.

Reply:
will be covered in #576